tsdb/querier.go

```diff
@@ -426,6 +426,16 @@ func labelNamesWithMatchers(r IndexReader, matchers ...*labels.Matcher) ([]strin
 	return r.LabelNamesFor(postings...)
 }
 
+// These are the things fetched when we move from one series to another.
```

Comment:
```suggestion
// seriesData are the data fetched when we move from one series to another.
```
hm... is this comment useful?

Reply:
I expanded it a little.